coordinator/src/services/request-builder.ts

```diff
@@ -0,0 +1,39 @@
+import Mustache from "mustache"
+import fs from "fs"
+import moment from "moment"
+import path from "path"
+import * as uuid from "uuid"
+
+const ebxmlRequestTemplate = fs.readFileSync(path.join(__dirname, "../resources/ebxml_request.mustache"), "utf-8")
+const cpaIdMap = new Map<string, string>(JSON.parse(process.env.CPA_ID_MAP))
+
+class EbXmlRequest {
+    timestamp = moment.utc().format()
+    conversation_id = uuid.v4().toUpperCase()
```

Comment:
Is the casing important for these? Or is it just a style thing?

Reply:
Named to match the values in the template which I stole from https://github.com/nhsconnect/integration-adaptor-mhs. Otherwise they'd be camel cased.

Reply:
I meant the fact that the GUID has been uppercased

Reply:
Ahh got it, yeah the MiM mandates that you use the uppercase representation of GUIDs.